pkg/config/config_windows.go

```diff
@@ -42,3 +49,15 @@ func osinit() {
 
 // NewAssetFs  Should never be called on non-android
 func setAssetFs(config Config) {}
+
+func init() {
```

Comment:
Is this overriding `init()` in config.go? Should these be in `osinit()` instead?

Reply:
It shouldn't override `init()` but you're absolutely right about moving the logic in here to `osinit()`. I think leaving it in `init()` may cause a race condition between the two `init()` functions in the config package. I will go ahead and move it.